omc3/correction/sequence_evaluation.py

```diff
@@ -0,0 +1,289 @@
+"""
+Similar to Sequence Parser but with MADX
+
+First: Set all variables to 0
+Then: Set one variable at a time to 1
+
+Compare results with case all==0
+
+"""
+import multiprocessing
+import os
+import pickle
```

Comment:
any updates on `feather` or `parquette`? pickle is problematic because it often requires the same packes to be installed (with compatible versions) as when the object was pickled.